src/ui/ui_review_menu_nbgl.c

```diff
@@ -149,45 +149,46 @@ static void prepareTxInfos(ui_approval_state_t state, bool data_warning) {
     memset(&infoLongPress, 0, sizeof(infoLongPress));
 
     txInfos.warnings[DATA_WARNING] = data_warning;
-    txInfos.flowTitle = "Review Transaction";
+    txInfos.flowTitle = "Review transaction";
     txInfos.rejectCb = (nbgl_callback_t) ui_callback_tx_cancel;
     txInfos.confirmCb = (nbgl_callback_t) ui_callback_tx_ok;
 
-    infoLongPress.text = "Confirm Transaction";
-    infoLongPress.longPressText = "Hold to confirm";
+    infoLongPress.text = "Sign transaction";
+    infoLongPress.longPressText = "Hold to sign";
     infoLongPress.icon = &C_stax_app_tron_64px;
 
     pairList.pairs = (nbgl_layoutTagValue_t *) txInfos.fields;
 
     switch (state) {
         case APPROVAL_TRANSFER:
+            PRINTF("TITI\n");
             txInfos.fields[0].item = "Amount";
             txInfos.fields[0].value = (const char *) G_io_apdu_buffer;
             txInfos.fields[1].item = "Token";
             txInfos.fields[1].value = fullContract;
             txInfos.fields[2].item = TRC20ActionSendAllow;
             txInfos.fields[2].value = toAddress;
-            txInfos.fields[3].item = "From Address";
+            txInfos.fields[3].item = "From";
```

Comment:
Could be interesting to use `#define` for all common strings ("From", "Review transaction", ...)

Reply:
Indeed. It won't save memory as the macros are replaced during pre-processing but still good practice.

Reply:
Using `static const char *` for common strings instead of macros.